include/SFML/System/Angle.hpp

```diff
@@ -0,0 +1,458 @@
+///
+/// \see degrees
+///
+////////////////////////////////////////////////////////////
+constexpr Angle radians(float angle);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of == operator to compare two angle values
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return True if both angle values are equal
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr bool operator ==(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of != operator to compare two angle values
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return True if both angle values are different
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr bool operator !=(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of < operator to compare two angle values
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return True if \a left is lesser than \a right
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr bool operator <(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of > operator to compare two angle values
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return True if \a left is greater than \a right
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr bool operator >(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of <= operator to compare two angle values
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return True if \a left is lesser or equal than \a right
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr bool operator <=(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of >= operator to compare two angle values
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return True if \a left is greater or equal than \a right
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr bool operator >=(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of unary - operator to negate an angle value
+///
+/// \param right Right operand (an angle)
+///
+/// \return Opposite of the angle value
```

Comment:
Maybe to not only mention the obvious, we could state that `-angle` represents a rotation in the opposite direction of `angle`? (I know most people are aware of this, but at least it's not implied by the operator itself).